canonical/lxd-ui
# All pr workflows in a single group

## Done

- group the pr checks into a single file for easier access under https://github.com/canonical/lxd-ui/actions

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - check all pr jobs are passing
